__tests__/app.test.js

```diff
@@ -111,4 +111,49 @@ describe("app", () => {
 
     })
 
+    describe("PATCH /api/articles/:article_id", () => {
+        test("Should return  article with updated votes from given article_id", () => {
+            return request(app).patch("/api/articles/1").send({ inc_votes: 10 }).expect(200).then(({ body }) => {
+                const { article } = body;
+                expect(article.article_id).toBe(1);
+                expect(article.votes).toBe(110);
+
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    body: expect.any(String),
+                });
+            });
+        })
+        test("should return 400 when no inc_votes is given", () => {
```

Comment:
I think that if no inc_votes were given i wouldn't say this is an error. This test could be modified though as inc_votes could be the wrong data type (ie str rather than num)

Reply:
added test `should return 400  when inc_votes is non-numeric`